rlutil.h

```diff
@@ -185,60 +185,71 @@ enum {
+ * ANSI_CURSOR_HIDE        - Hides the cursor
+ * ANSI_CURSOR_SHOW        - Shows the cursor
+ * ANSI_BLACK              - Black
+ * ANSI_RED                - Red
+ * ANSI_GREEN              - Green
+ * ANSI_BROWN              - Brown / dark yellow
+ * ANSI_BLUE               - Blue
+ * ANSI_MAGENTA            - Magenta / purple
+ * ANSI_CYAN               - Cyan
+ * ANSI_GREY               - Grey / dark white
+ * ANSI_DARKGREY           - Dark grey / light black
+ * ANSI_LIGHTRED           - Light red
+ * ANSI_LIGHTGREEN         - Light green
+ * ANSI_YELLOW             - Yellow (bright)
+ * ANSI_LIGHTBLUE          - Light blue
+ * ANSI_LIGHTMAGENTA       - Light magenta / light purple
+ * ANSI_LIGHTCYAN          - Light cyan
+ * ANSI_WHITE              - White (bright)
+ * ANSI_BACKGROUND_BLACK   - Black background
+ * ANSI_BACKGROUND_RED     - Red background
+ * ANSI_BACKGROUND_GREEN   - Green background
+ * ANSI_BACKGROUND_YELLOW  - Yellow background
+ * ANSI_BACKGROUND_BLUE    - Blue background
+ * ANSI_BACKGROUND_MAGENTA - Magenta / purple background
+ * ANSI_BACKGROUND_CYAN    - Cyan background
+ * ANSI_BACKGROUND_WHITE   - White background
  */
-const RLUTIL_STRING_T ANSI_CLS                     = "\033[2J";
-const RLUTIL_STRING_T ANSI_BLACK                   = "\033[22;30m";
-const RLUTIL_STRING_T ANSI_RED                     = "\033[22;31m";
-const RLUTIL_STRING_T ANSI_GREEN                   = "\033[22;32m";
-const RLUTIL_STRING_T ANSI_BROWN                   = "\033[22;33m";
-const RLUTIL_STRING_T ANSI_BLUE                    = "\033[22;34m";
-const RLUTIL_STRING_T ANSI_MAGENTA                 = "\033[22;35m";
-const RLUTIL_STRING_T ANSI_CYAN                    = "\033[22;36m";
-const RLUTIL_STRING_T ANSI_GREY                    = "\033[22;37m";
-const RLUTIL_STRING_T ANSI_DARKGREY                = "\033[01;30m";
-const RLUTIL_STRING_T ANSI_LIGHTRED                = "\033[01;31m";
-const RLUTIL_STRING_T ANSI_LIGHTGREEN              = "\033[01;32m";
-const RLUTIL_STRING_T ANSI_YELLOW                  = "\033[01;33m";
-const RLUTIL_STRING_T ANSI_LIGHTBLUE               = "\033[01;34m";
-const RLUTIL_STRING_T ANSI_LIGHTMAGENTA            = "\033[01;35m";
-const RLUTIL_STRING_T ANSI_LIGHTCYAN               = "\033[01;36m";
-const RLUTIL_STRING_T ANSI_WHITE                   = "\033[01;37m";
-const RLUTIL_STRING_T ANSI_BACKGROUND_BLACK        = "\033[40m";
-const RLUTIL_STRING_T ANSI_BACKGROUND_RED          = "\033[41m";
-const RLUTIL_STRING_T ANSI_BACKGROUND_GREEN        = "\033[42m";
-const RLUTIL_STRING_T ANSI_BACKGROUND_BROWN        = "\033[43m";
-const RLUTIL_STRING_T ANSI_BACKGROUND_BLUE         = "\033[44m";
-const RLUTIL_STRING_T ANSI_BACKGROUND_MAGENTA      = "\033[45m";
-const RLUTIL_STRING_T ANSI_BACKGROUND_CYAN         = "\033[46m";
-const RLUTIL_STRING_T ANSI_BACKGROUND_GREY         = "\033[47m";
-// Remaining colors not supported
+#if __APPLE__
+const RLUTIL_STRING_T ANSI_CLS                = "\033[0m\033[2J";
```

Comment:
The first two characters are for `iTerm` which do not support `"\033c"`, but the system's default `Terminal.app` supports it, so it should be used to reset the screen instead of clearing it the old way.